### ISSUE TITLE
Possible bug in RenderFlow.paintChild()

### ISSUE THREAD

**Issue by schwa423** [flutter/flutter#5634]
This is a noticed-during-drive-by, purely-by-inspection report.

In the 0.0 < opacity < 1.0 case, we see:

```
_paintingContext.pushOpacity(_paintingOffset, _getAlphaFromOpacity(opacity), (PaintingContext context, Offset offset) {
    _paintingContext.pushTransform(needsCompositing, offset, transform, painter);
});
```

I would expect to see `context.pushTransform()` instead of `_paintingContext.pushTransform()`.

If the current code is correct, it probably warrants a comment, because this is confusing to me.

**Comment by abarth** [comment]
Yes, looks like a bug.  I suspect it won't make any difference in practice as the two objects will always happen to be the same, but for cleanliness we should make the change you suggest.  Thanks!

**Comment by SUPERETDUPER** [comment]
@Hixie Seems like this was fixed with #15543
